docs/kpod-umount.1.md

```diff
@@ -0,0 +1,19 @@
+% kpod(1) kpod-umount - Unmount a working container's root filesystem.
+% Dan Walsh
+# kpod-umount "1" "July 2017" "kpod"
+
+## NAME
+kpod umount - Unmount a working container's root file system.
```

Comment:
No each mount is an additional mount.  Think of this as doing 
```
mount -o bind /tmp /mnt
mount -o bind /tmp /mnt
umount /mnt
```
I think  if you attempted to umount a running container it will fail, because the mount point is busy.  There are some potential corner cases where you could modify content from the outside and cause the container process to die, but this is the same as if you docker exec into a container and modified the content, processes could die.